src/Reference/Task5.java

```diff
@@ -0,0 +1,45 @@
+package Reference;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.util.Calendar;
+import java.util.Date;
+
+public class Task5 {
+
+    public static void main(String[] args) {
+
+        LocalDate date = LocalDate.now();
+        DayOfWeek dayOfWeek = date.getDayOfWeek();
+
+        switch (dayOfWeek){
```

Comment:
could you add a solution without using a switch case ?

Reply:
System.out.println(date.with(TemporalAdjusters.next(DayOfWeek.FRIDAY)));
System.out.println(date.with(TemporalAdjusters.previous(DayOfWeek.FRIDAY)));